### PR TITLE
Create teast-util renderWithProwiders and fix existing imports. Tests for BestSellers

### LINKED ISSUE #63: Create test-util RenderWithProviders
To avoid duplication of code in each test file, add RenderWithProviders test-utils that simplifies the rendering of components with all necessary providers. 
This utility ensures that all required contexts, such as Redux, React Router, and any other global providers, are consistently applied across your test suite. Here's how you can implement and use RenderWithProviders

### PR DESCRIPTION
**Pull request by OliaHrytsak**
image
image

Stryker coverage
image

image

### CODE REVIEW

File: code/src/layouts/best-sellers/BestSellers.test.tsx [hunk 1]
```diff
@@ -0,0 +1,73 @@
+import { screen } from "@testing-library/react";
+
+import BestSellers from "@/layouts/best-sellers/BestSellers";
+import { useGetProductsQuery } from "@/store/api/productsApi";
+import { renderWithProviders } from "@/utils/test-utils";
+
+import { mockData } from "@/layouts/best-sellers/BestSellers.constants";
+
+jest.mock("../../store/api/productsApi", () => ({
```
Comment (D1nviz) [inline code/src/layouts/best-sellers/BestSellers.test.tsx]: can we use alias here?

**Comment by D1nviz** [comment]
Do we need these 2 files? maybe would be better to put them to git ignore?
image

File: code/config/jest.polyfills.js [hunk 2]
```diff
@@ -1,2 +1,3 @@
 // Canvas
+// import "node-fetch";
```
Comment (heorhiiss) [inline code/config/jest.polyfills.js]: What does this comment mean?

**Review by mhevyk — APPROVED** [review]

File: code/src/layouts/best-sellers/BestSellers.test.tsx (hunk 1 above)
Comment (OliaHrytsak) [reply code/src/layouts/best-sellers/BestSellers.test.tsx]: Yes. Fixed. Thank you.

File: code/config/jest.polyfills.js (hunk 2 above)
Comment (OliaHrytsak) [reply code/config/jest.polyfills.js]: I fogot to remove. Fixed. Thaak you

**Comment by OliaHrytsak** [comment]
> Do we need these 2 files? maybe would be better to put them to git ignore? image

I dont know actually, because we have this files on our develop branch with initial data

**Review by heshvab — APPROVED** [review]

**Review by dmtrth25 — APPROVED** [review]

**Review by D1nviz — APPROVED** [review]

**Pull request merged**